app/services/remove_status_service.rb

```diff
@@ -12,6 +12,7 @@ class RemoveStatusService < BaseService
   # @option  [Boolean] :immediate
   # @option  [Boolean] :preserve
   # @option  [Boolean] :original_removed
+  # @option  [Boolean] :deliver_quietly
```

Comment:
Maybe call it something like `skip_redis`?

Also I think instead of having the check in every method, I'd have the method calls grouped under a `unless skip_redis?` condition.

Reply:
I went with doing it in each method as each method might perform additional logic we do want executing, for instance, `remove_from_hashtags` decrements some counters, and we still want that to happen.

I wasn't sure if something like `skip_redis` was clear enough? But it's easy enough to change if we want a different name for it.

Reply:
> I went with doing it in each method as each method might perform additional logic we do want executing, for instance, remove_from_hashtags decrements some counters, and we still want that to happen.

You're right, I overlooked that.

Reply:
What about `delete_quietly` instead of `deliver_quietly` ? Entirely up to you, I personally don't think there is a good name for this option.

Reply:
`DeleteAccountService` has `skip_side_effects` and `skip_activitypub`, so I think `skip_redis` makes sense here, if a bit low-level.

Reply:
Maybe `skip_realtime` or `skip_streaming` would make most sense whilst not being `skip_redis` specifically.

i think I like `skip_streaming` the best.

Reply:
I'm fine with either `skip_streaming` or `skip_redis`.